newsletter/backend/newsletters/tests/test_models.py

```diff
@@ -0,0 +1,15 @@
+from newsletters.models import Agreements
+
+
+def test_representation_in_gui():
```

Comment:
Also I would add model name in this test name like `test_agreements_gui_representation`. If it fails in CI/CD we won't know which gui representation failed. 😊

Reply:
Done.